source/base/Allocator.ooc

```diff
@@ -0,0 +1,25 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+import Owner
+
+AbstractAllocator: abstract class {
```

Comment:
Good idea. I agree that allocating allocators for every object seems unnecessary. 

I shall add default allocators now to prevent this.